src/scss/_base.scss

```diff
@@ -6,9 +6,13 @@ body {
 }
 
 .h-80vh {
-  height: 80vh;
+  height: 50vh;
 }
 
 .color-accent {
   color: $color-accent;
 }
+
+.text-0 {
+  font-size: 6rem;
+}
```

Comment:
Use [RTS](https://getbootstrap.com/docs/5.2/getting-started/rfs/). Move to `_fonts.scss`.